generators/server/templates/pom.xml.ejs

```diff
@@ -163,7 +163,7 @@
             <dependency>
                 <groupId>io.github.jhipster</groupId>
                 <artifactId>jhipster-dependencies</artifactId>
-                <version>${jhipster-dependencies.version}</version>
```

Comment:
I find this misleading to use "jhipster-framework" and not "jhipster-dependencies" for the variable name, I'm going to revert and use "jhipster-dependencies" instead - I hope that's fine with you